internal/validation/validate_test.go

```diff
@@ -953,44 +945,7 @@ var _ = Describe("Validate function", func() {
 		}).Validate(context.TODO(), fakeClient, input, networkingv1beta1.VirtualServiceList{})
 
 		//then
-		Expect(problems).To(HaveLen(1))
-		Expect(problems[0].AttributePath).To(Equal(".spec.service"))
-		Expect(problems[0].Message).To(Equal("No label selectors found for service"))
-	})
-
-	It("Should fail with service on path level without labels selector", func() {
```

Comment:
you remove on path level validate test, that is why you did not cover case / usage of no label selector on a service used on path level.

Reply:
I re-added the test https://github.com/kyma-project/api-gateway/pull/402/commits/b4f7dca6dc52fa9fdaf0bb43f39f2cacd7366db0